.travis.yml

```diff
@@ -10,15 +10,13 @@ php:
   - 7.0
   - 7.1
 
-  matrix:
+matrix:
   fast_finish: true
   include:
```

Comment:
You need hhvm.php7.all=true in the ini file; I don't know how to do this on Travis. Composer might also need some poking to understand what that does.

cc @paulbiss, @aorenste for that - also, it might be time to flip the default of that ini setting.